gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/UserServiceImpl.java

```diff
@@ -292,8 +292,8 @@ public Completable resetPassword(Domain domain, String userId, String password,
                                             user.setUpdatedAt(new Date());
                                             return userService.update(user);
                                         })
-                                        .doOnSuccess(user1 -> auditService.report(AuditBuilder.builder(UserAuditBuilder.class).principal(principal).type(EventType.USER_PASSWORD_RESET).user(user)))
-                                        .doOnError(throwable -> auditService.report(AuditBuilder.builder(UserAuditBuilder.class).principal(principal).type(EventType.USER_PASSWORD_RESET).throwable(throwable)));
+                                        .doOnSuccess(user1 -> auditService.report(AuditBuilder.builder(UserAuditBuilder.class).client(user.getClient()).principal(principal).type(EventType.USER_PASSWORD_RESET).user(user)))
+                                        .doOnError(throwable -> auditService.report(AuditBuilder.builder(UserAuditBuilder.class).client(user.getClient()).principal(principal).type(EventType.USER_PASSWORD_RESET).user(user).throwable(throwable)));
```

Comment:
Replace `client(user.getClient())` with `client(client)` to get all the application information.

Reply:
FP